### PR TITLE
Administrative assistant

### PR DESCRIPTION
**Pull request by Radezolid**
## About the PR
This role is meant for people that wish to learn how command works while also giving room for good RP, assisting with paperwork and any other general thing a command member needs help with.
They get command access and just command + common radio.
They don't need any mapping besides the spawner itself since they don't get an office and everything they will need they can get it through loadouts.
This PR it's a continuation of https://github.com/DeltaV-Station/Delta-v/pull/1207
## Why / Balance
Learner role + more RP focused command member.

## Technical details
Added a new headset, ID, stamp, clothes specific to the assistant, made new loadouts for the wintercoats to be referenced in future PRs.

## Media

Loadout:

<details>
<summary> Jumpsuit </summary>

image

</details>
<details>
<summary> Outer clothes </summary>

image

</details>
<details>
<summary> Shoes </summary>

image

</details>
<details>
<summary> Headset </summary>

image

</details>
<details>
<summary> Gloves </summary>

image

</details>
<details>
<summary> Backpack </summary>

image

</details>
<details>
<summary> Glasses </summary>

image

</details>

Items they will have round start without touching loadouts:

image

## Requirements

- [x] I have tested all added content and changes.
- [x] I have added media to this PR or it does not require an ingame showcase.

## Breaking changes
Nothing.

**Changelog**

No CL, one should be done when the spawners are mapped.

### CODE REVIEW

**Comment by Radezolid** [comment]
Having a small issue rn, that it doesn't let me check the loadouts, i am missing something and i'm not seeing what.
image
Any ideas?

**Comment by Radezolid** [comment]
Hey, @deltanedas any idea what might be the cause of this?

> Having a small issue rn, that it doesn't let me check the loadouts, i am missing something and i'm not seeing what. image

**Comment by deltanedas** [comment]
the RoleLoadoutPrototype has to have the id `JobAdministrativeAssistant`

**Comment by Radezolid** [comment]
> the RoleLoadoutPrototype has to have the id `JobAdministrativeAssistant`

Thanks

**Comment by Lyndomen** [comment]
3 days direction review

File: Resources/Prototypes/DeltaV/Entities/Clothing/Uniforms/jumpskirts.yml [hunk 1]
```diff
@@ -189,7 +189,7 @@
     sprite: DeltaV/Clothing/Uniforms/Jumpskirt/cj.rsi
   - type: Clothing
     sprite: DeltaV/Clothing/Uniforms/Jumpskirt/cj.rsi
-    
+
```
Comment (Lyndomen) [inline Resources/Prototypes/DeltaV/Entities/Clothing/Uniforms/jumpskirts.yml]: le whitespace

File: Resources/Prototypes/DeltaV/Entities/Markers/Spawners/jobs.yml [hunk 2]
```diff
@@ -105,3 +105,16 @@
...
+  id: SpawnPointAdminAssistant
+  name: administrative assistant
+  components:
+  - type: SpawnPoint
+    job_id: AdministrativeAssistant
+  - type: Sprite
+    layers:
+      - state: green
+      - sprite: DeltaV/Markers/jobs.rsi
+        state: adminassistant
```
Comment (deltanedas) [inline Resources/Prototypes/DeltaV/Entities/Markers/Spawners/jobs.yml]: unindent

File: Resources/Prototypes/DeltaV/Roles/Jobs/Command/administrative_assistant.yml [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+      time: 10800 # 3 hours
+    - !type:DepartmentTimeRequirement
+      department: Security
+      time: 10800 # 3 hours
+    - !type:DepartmentTimeRequirement
+      department: Justice
+      time: 10800 # 3 hours
+    - !type:DepartmentTimeRequirement
+      department: Epistemics
+      time: 10800 # 3 hours
```
Comment (deltanedas) [inline Resources/Prototypes/DeltaV/Roles/Jobs/Command/administrative_assistant.yml]: unindent

File: Resources/Prototypes/DeltaV/Roles/Jobs/Command/administrative_assistant.yml [hunk 4]
```diff
@@ -0,0 +1,51 @@
...
+  startingGear: AdminAssistantGear
+  icon: "JobIconAdminAssitant"
+  supervisors: job-supervisors-command
+  canBeAntag: false
+  access:
+    - Command
+    - Maintenance
```
Comment (deltanedas) [inline Resources/Prototypes/DeltaV/Roles/Jobs/Command/administrative_assistant.yml]: .

File: Resources/Prototypes/DeltaV/Roles/Jobs/Command/administrative_assistant.yml [hunk 5]
```diff
@@ -0,0 +1,51 @@
...
+  special:
+    - !type:AddImplantSpecial
+      implants: [MindShieldImplant]
+    - !type:AddComponentSpecial
+      components:
+      - type: CommandStaff
```
Comment (deltanedas) [inline Resources/Prototypes/DeltaV/Roles/Jobs/Command/administrative_assistant.yml]: .

File: Resources/Prototypes/Roles/Jobs/departments.yml [hunk 6]
```diff
@@ -80,6 +80,7 @@
   - ERTEngineer
   - DeathSquad
   - StationAi # DeltaV - added for playtime trackers
+  #- AdministrativeAssistant # Delta V - Uncomment once ready to deploy. Administrative Assistant, see Resources/Prototypes/Nyanotrasen/Roles/Jobs/Command/admin_assistant.yml
```
Comment (deltanedas) [inline Resources/Prototypes/Roles/Jobs/departments.yml]: nyano :trollface:

File: Resources/Textures/DeltaV/Clothing/Uniforms/Jumpskirt/admin_assistant.rsi/meta.json [hunk 7]
```diff
@@ -0,0 +1,30 @@
+{
```
Comment (deltanedas) [inline Resources/Textures/DeltaV/Clothing/Uniforms/Jumpskirt/admin_assistant.rsi/meta.json:1]: 4 space indent

File: Resources/Textures/DeltaV/Clothing/Uniforms/Jumpsuit/admin_assistant.rsi/meta.json [hunk 8]
```diff
@@ -0,0 +1,30 @@
+{
```
Comment (deltanedas) [inline Resources/Textures/DeltaV/Clothing/Uniforms/Jumpsuit/admin_assistant.rsi/meta.json:1]: .

File: Resources/Textures/DeltaV/Objects/Misc/bureaucracy.rsi/meta.json [hunk 9]
```diff
@@ -1,14 +1,14 @@
 {
```
Comment (deltanedas) [inline Resources/Textures/DeltaV/Objects/Misc/bureaucracy.rsi/meta.json]: untouch this file its unrelated

**Review by deltanedas — CHANGES_REQUESTED** [review]

File: Resources/Textures/DeltaV/Clothing/Uniforms/Jumpskirt/admin_assistant.rsi/meta.json (hunk 7 above)
Comment (Radezolid) [reply Resources/Textures/DeltaV/Clothing/Uniforms/Jumpskirt/admin_assistant.rsi/meta.json:1]: Forgot to check indents on the meta.json, thanks.

File: Resources/Textures/DeltaV/Objects/Misc/bureaucracy.rsi/meta.json (hunk 9 above)
Comment (Radezolid) [reply Resources/Textures/DeltaV/Objects/Misc/bureaucracy.rsi/meta.json]: The stamp was there previously until i learned it's hardcoded to check on the upstream files.

**Review by deltanedas — DISMISSED** [review]

**Review by Lyndomen — DISMISSED** [review]
Approved for a test, we will be gathering feedback. I'm personally mostly concerned about the playtime reqs causing this role to not be accessible enough to serve as a learner role, but time will tell. 

Side question, not sure if other roles have a guidebook entry on their role, but I think it would be nice to see for all DeltaV exclusive roles

**Comment by Radezolid** [comment]
> Approved for a test, we will be gathering feedback. I'm personally mostly concerned about the playtime reqs causing this role to not be accessible enough to serve as a learner role, but time will tell.
> 
> Side question, not sure if other roles have a guidebook entry on their role, but I think it would be nice to see for all DeltaV exclusive roles

I can modify the time requirements, what times do you have in mind?

**Comment by Radezolid** [comment]
Hey @deltanedas whenever you can merge this please.

**Review by Lyndomen — COMMENTED** [review]
25 hour overall playtime would be my best. That way people can actually unlock it before they unlock CE or mysta or whatever

**Comment by Radezolid** [comment]
> 25 hour overall playtime would be my best. That way people can actually unlock it before they unlock CE or mysta or whatever

Just 25 hours, no departamental time?

**Review by Lyndomen — APPROVED** [review]

**Review by deltanedas — APPROVED** [review]

**Pull request merged**